providers.tf

```diff
@@ -0,0 +1,4 @@
+provider "datadog" {
+  api_key = local.secrets.datadog_api_key
+  app_key = local.secrets.datadog_app_key
+}
```

Comment:
I say we remove this file entirely since we shouldn't have a providers.tf in a child module.